File: Task 2/avNum.js
// variables
let num_1 = Number(prompt("pick a number:"));
let num_2 = Number(prompt("pick another number:"));
let num_3 = Number(prompt("pick a third number:"));
let avNum = (num_1 + num_2 + num_3)/3
alert(`The average of your chosen numbers is ${avNum}`)


// functions
function num_av() {
    let num_1 = prompt("pick a number: ");
    let num_2 = prompt("pick another number: ");
    let num_3 = prompt("pick a third number");
    let avNum = (Number(num_1) + Number(num_2) + Number(num_3))/3
    return `The average of your chosen numbers is ${avNum}`
}

alert(num_av())

// or (without input from the user)

function n_av(_num1, _num2, _num3) {
    return (_num1 + _num2 + _num3)/3
}

console.log(n_av(3, 3, 3))

// arrow function for the 2nd function solution which can also be used for the 1st. this is just rewriting the function. both works

// const n_av = (_num1, _num2, _num3) => (_num1 + _num2 + _num3)/3;

// console.log(n_av(3, 3, 3));

File: Task 2/avNum.py
# variables
num_1 = float(input("pick a number: "))
num_2 = float(input("pick another number: "))
num_3 = float(input("pick a third number: "))
avNum = (num_1 + num_2 + num_3)/3
print("The average of your chosen numbers is {}".format(avNum))


# functions
def num_av():
    num_1 = float(input("pick a number: "))
    num_2 = float(input("pick another number: "))
    num_3 = float(input("pick a third number: "))
    avNum = (num_1 + num_2 + num_3)/3
    return avNum

print("The average of your chosen numbers is {}".format(avNum))

def n_av(n_1, n_2, n_3):
    av_num = (n_1 + n_2 + n_3)/3
    return av_num

print(n_av(3, 3, 4))
